meteor/sofie-server/data-access/repositories/mongo/mongo-entity-converter.ts

```diff
@@ -43,7 +43,7 @@ export interface MongoRundown {
 	showStyleVariantId: string
 	showStyleBaseId: string
 	modified: number
-	isActive: boolean
+	isActive?: boolean // TODO: Remove optionality when we have control over data structure.
```

Comment:
I would say no need to make it optional at all since this is all contained within `convertToRundown`.

Reply:
That seems wrong. In `convertToRundown`, you would probably use nullish coalescing on a non-nullable value (per type). By doing it on the model, we describe the actual data structure that we work on.